### PR TITLE
Update Crowdloan Page

### PR DESCRIPTION
**Pull request by salmad3**
- use inject value for min crowdloan param
- general edits
- part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

**Comment by salmad3** [comment]
Pending: https://github.com/w3f/polkadot-wiki/pull/3167

File: docs/learn/learn-crowdloans.md [hunk 1]
```diff
@@ -7,13 +7,22 @@ slug: ../learn-crowdloans
 ---
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} allows parachains to source tokens
-for their parachain bids in a decentralised crowdloan. If you are here for guidance on how to 
+for their parachain bids in a decentralised crowdloan. 
+
+:::note Contributing to a crowdloan
+If you are here for guidance on how to
```
Comment (profradha) [inline docs/learn/learn-crowdloans.md]:
```suggestion
:::note Contributing to a crowdloan

If you are here for guidance on how to 
```

File: docs/learn/learn-crowdloans.md [hunk 2]
```diff
@@ -52,24 +61,28 @@ runtime upgrades (as determined through its own local governance).
...
 
-> {{ polkadot: **Important**: The minimum balance for contributions for a crowdloan campaign is 
-> currently set to [5 DOTs](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L1267). 
-> This is in an attempt to make crowdloans as accessible as possible while maintaining a 
-> balance to justify the use of the network's resources. :polkadot }}{{ kusama: 
-> **Important**: The minimum balance for contributions for a crowdloan campaign is currently set to
-> [0.1 KSM](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L1218). 
-> This is in an attempt to make crowdloans as accessible as possible while maintaining a balance to 
-> justify the use of the network's resources. :kusama }}.
+:::info minimum balance
```
Comment (profradha) [inline docs/learn/learn-crowdloans.md]:
```suggestion
:::info Minimum Crowdloan Contribution
```

File: docs/learn/learn-crowdloans.md [hunk 3]
```diff
@@ -52,24 +61,28 @@ runtime upgrades (as determined through its own local governance).
...
 Each created campaign will have an index. Once a crowdloan campaign is open, anyone can participate
 by sending a special transaction that references the campaign's index. Tokens used to participate
 must be transferable &mdash; that is, not locked for any reason, including staking, vesting, and
 governance &mdash; because they will be moved into a module-controlled account that was generated
 uniquely for this campaign.
 
-> **Important**: All crowdloan contributions are handled by the Crowdloan module’s logic where a
-> campaign is identified by an index, not by address. **Never transfer tokens to an address in 
-> support of a campaign.**
+:::caution A campaign is identified by an index, not by address
```
Comment (profradha) [inline docs/learn/learn-crowdloans.md]:
```suggestion
:::caution Do not send Crowdloan contributions directly to the Parachain address
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**